src/messages.ts

```diff
@@ -24,6 +24,8 @@ export const FAILURE_MESSAGES = {
   DATABASE_ERROR: 'Database error',
   UNEXPECTED_ERROR: 'Unexpected error',
 
+  FILE_NOT_IMAGE: 'The file is not an image',
```

Comment:
Would `FILE_IS_NOT_IMAGE` fit better ? We could think that we want a file and not an image.